retention/expiry.go

```diff
@@ -417,59 +421,6 @@ func ExpireOnS3(ctx context.Context, s3ControlClient s3controliface.S3ControlAPI
 			}
 			taggedBuckets[done.bucketName] = struct{}{}
 		}
-
```

Comment:
is this part of this PR?

Reply:
this is outdated commit - the latest version should include just the lint changes based on merge with master